helper/RPC.js

```diff
@@ -0,0 +1,36 @@
+const axios = require('axios');
+const { RPC_URLS } = require('../config/RPCConfig');
+
+const buildURL = (ip) => `http://${ip}:7777/rpc`;
+
+const randomURLIndex = (rpcList) => Math.floor(Math.random() * rpcList.length);
+
+const getRPCURL = async (rpcURLList = RPC_URLS) => {
```

Comment:
We can improve it later by ping the nodes till having the success response by using retry package.